ansible/memcached-operator/README.md

```diff
@@ -28,75 +29,76 @@ $ cd operator-sdk-samples/ansible/memcached-operator
 
-NAME                                 TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
-service/memcached-operator-metrics   ClusterIP   10.98.192.187   <none>        8383/TCP   31s
+NAME                                                            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+service/memcached-operator-controller-manager-metrics-service   ClusterIP   10.102.107.68   <none>        8443/TCP   19s
 
-NAME                                          READY   UP-TO-DATE   AVAILABLE   AGE
-deployment.apps/example-memcached-memcached   3/3     3            3           22s
-deployment.apps/memcached-operator            1/1     1            1           36s
+NAME                                                    READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/memcached-operator-controller-manager   1/1     1            1           19s
+deployment.apps/memcached-sample-memcached              1/1     1            1           12s
 
-NAME                                                    DESIRED   CURRENT   READY   AGE
-replicaset.apps/example-memcached-memcached-b885dcc75   3         3         3       22s
-replicaset.apps/memcached-operator-df88b85f7            1         1         1       36s
+NAME                                                              DESIRED   CURRENT   READY   AGE
+replicaset.apps/memcached-operator-controller-manager-f896cd75b   1         1         1       19s
+replicaset.apps/memcached-sample-memcached-6456bdd5fc             1         1         1       12s
 ```
 
-### Uninstalling
+### Cleanup
 
-To uninstall all that was performed in the above step run `make uninstall`.
+To leave the operator, but remove the memcached sample pods, delete the
+CR.
 
-### Troubleshooting
+```sh
+kubectl delete -f config/samples/cache_v1alpha1_memcached.yaml -n memcached-operator-system
+```
 
-Use the following command to check the operator logs.
+To clean up everything:
 
+```sh
+make undeploy
 ```
-kubectl logs deployment.apps/memcached-operator -n memcached
-```
+### Troubleshooting
 
-**NOTE:** This project is configured with the environment variable `ANSIBLE_DEBUG_LOGS` as `True`, however, note that it is `False` by default.
+Run the following command to check the operator logs.
+
+```sh
+kubectl logs deployment.apps/memcached-operator-controller-manager -n memcached-operator-system -c manager
+```
 
-**NOTE** To have further information about how to develop Ansible operators with [Operator-SDK][operator_sdk] check the [Ansible User Guide for Operator-SDK][ansible-guide]
+### Extras
 
-### Testing the Operator
+This project was created by using the [gen-ansible-memcached.sh][gen-ansible-memcached.sh] script.
 
-See [Testing Ansible Operators with Molecule][ansible-test-guide] documentation to know how to use the operator framework features to test it.  
+For mor information see the [Ansible-based operator docs][ansible-docs].
```

Comment:
WDYT about : 

Also, you can start with Ansible-based Operators with SDK by checking its [quickstart][ansible_guide]. -> and point out to Ansible quick start. It will redirect the user to the tutorial but give the opportunity for new users to just run a few commands to check its working as well.

Reply:
That works-- but I thought we could just link to the top level. That way we don't have to guess how they want to proceed.

Reply:
Maybe you are right .. To know more about Ansible-based Operators with SDK see [here][ansible_menu]?
In will do == in Helm. .. just trying to reach the consensus :-)

Reply:
I will use it `For mor information see the [Ansible-based operator docs][ansible-docs].` too 👍